guides/source/active_record_validations.md

```diff
@@ -435,7 +435,7 @@ The default error message for this helper is _"is not included in the list"_.
 
 ### `length`
 
-This helper validates the length of the attributes' values. It provides a
+This helper validates the length of the attribute's value. It provides a
```

Comment:
I think plural was correct here. You can pass more than one attribute

Reply:
amended